prov/gni/include/fi_ext_gni.h

```diff
@@ -54,6 +54,8 @@ typedef enum dom_ops_val { GNI_MSG_RENDEZVOUS_THRESHOLD,
 			   GNI_MBOX_MSG_MAX_SIZE,
 			   GNI_RX_CQ_SIZE,
 			   GNI_TX_CQ_SIZE,
+			   GNI_MAX_RETRANSMITS,
+			   GNI_ERR_INJECT_COUNT,
```

Comment:
Is GNI_ERR_INJECT_COUNT primarily for testing purposes?  If so, I'd leave it out of the man page.  We can discuss later if we want to compile it out for "production" builds.